example/pages/index.js

```diff
@@ -97,69 +95,76 @@ export default function Home() {
 
           <p>Install the package by running:</p>
 
-          <pre><code>{`yarn add next-cloudinary
+          <pre>
+            <code>{`yarn add next-cloudinary
 # or
-npm install next-cloudinary`}</code></pre>
+npm install next-cloudinary`}</code>
+          </pre>
 
-          <p>Add an environment variable inside of your `.env.local`, `.env`, or deployment platform:</p>
+          <p>
+            Add an environment variable inside of your `.env.local`, `.env`, or
+            deployment platform:
+          </p>
 
-          <pre><code>{`NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME="<Your Cloud Name>"`}</code></pre>
+          <pre>
+            <code>{`NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME="<Your Cloud Name>"`}</code>
+            <code>{`NEXT_PUBLIC_CLOUDINARY_UPLOAD_PRESET="<Your Preset>"`}</code>
```

Comment:
i think it would make sense to include the additional environment variables along with the Upload Widget documentation instead of including it here, since it adds an extra unnecessary step if someone only wants to use the image

Reply:
Yes will remove this and just add to documentation instead